pynta/utils.py

```diff
@@ -24,19 +24,19 @@ def get_unique_sym(geoms):
 
     '''
     comparator = SymmetryEquivalenceCheck()
-
+    geoms_copy = deepcopy(geoms)
```

Comment:
I believe this new copying and indexing of the original list of geoms is not necessary for the uniqueness functions that operate on paths.